src/main/java/it/pagopa/interop/probing/eservice/operations/util/EnumUtilities.java

```diff
@@ -35,18 +35,28 @@ public List<String> convertListFromMonitorToPdnd(List<EserviceMonitorState> stat
 
   public EserviceMonitorState fromPdndToMonitorState(EserviceView view) {
     return switch (view.getState()) {
-      case ACTIVE -> checkND(view) ? EserviceMonitorState.N_D : EserviceMonitorState.ONLINE;
+      case ACTIVE -> checkND(view) ? EserviceMonitorState.N_D
+          : checkOFFLINE(view) ? EserviceMonitorState.OFFLINE : EserviceMonitorState.ONLINE;
       case INACTIVE -> checkND(view) ? EserviceMonitorState.N_D : EserviceMonitorState.OFFLINE;
       default -> throw new IllegalArgumentException("Invalid state {}= " + view.getState());
     };
   }
 
+  public boolean fromEnumToBoolean(EserviceInteropState viewState) {
```

Comment:
as someone that could use methods in this class, it's not clear what an api with signature `boolean EnumUtilities.fromEnumToBoolean(EserviceInteropState viewState)` does.

Maybe 
```suggestion
  public boolean isActive(EserviceInteropState viewState) {
```
?

Reply:
ok